src/main/kotlin/link/infra/packwiz/installer/UpdateManager.kt

```diff
@@ -59,7 +61,7 @@ class UpdateManager internal constructor(private val opts: Options, val ui: IUse
 		checkOptions()
 
 		ui.submitProgress(InstallProgress("Loading manifest file..."))
-		val gson = GsonBuilder().registerTypeAdapter(Hash::class.java, Hash.TypeHandler()).create()
+		val gson = GsonBuilder().registerTypeAdapter(Hash::class.java, Hash.TypeHandler()).setPrettyPrinting().create()
```

Comment:
This increases the filesize quite a bit for large packs, not sure if it's necessary though it does make it easier to debug :P

Reply:
Yeah, please do :)